<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#121212;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:#272727;
	color:#fff;
	border-radius:3px;
}

QPushButton:hover{
	background-color:#383838;
}

QPushButton:pressed{
	background-color:#272727;
}</string>
    </property>
    <property name="text">
     <string>All Trackers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBestIP">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:#272727;
	color:#fff;
	border-radius:3px;
}

QPushButton:hover{
	background-color:#383838;
}

QPushButton:pressed{
	background-color:#272727;
}</string>
    </property>
    <property name="text">
     <string>Best Trackers IP</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edtField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>711</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;</string>
    </property>
    <property name="text">
     <string>Made by DarkCeptor44</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBest">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:#272727;
	color:#fff;
	border-radius:3px;
}

QPushButton:hover{
	background-color:#383838;
}

QPushButton:pressed{
	background-color:#272727;
}</string>
    </property>
    <property name="text">
     <string>Best Trackers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAllIP">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:#272727;
	color:#fff;
	border-radius:3px;
}

QPushButton:hover{
	background-color:#383838;
}

QPushButton:pressed{
	background-color:#272727;
}</string>
    </property>
    <property name="text">
     <string>All Trackers IP</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
